sips/sip-028/sip-028-sbtc_peg.md

```diff
@@ -0,0 +1,217 @@
+- Bitcoin can be converted into sBTC within 3 Bitcoin blocks; and sBTC can be converted into Bitcoin within 6 Bitcoin blocks. sBTC relies on the forking behavior guaranteed by [SIP-021](https://github.com/stacksgov/sips/blob/feat/sip-021-nakamoto/sips/sip-021/sip-021-nakamoto.md) in order to maintain the peg wallet correctly across forks.
+
+## Specification
+
+Management of the sBTC peg wallet on the Bitcoin blockchain is decentralized, involving the sBTC Signer Set rather than a single custodian. At launch, the sBTC protocol will be maintained by 15 independent entities that make up the sBTC Signer Set. The eligibility criteria to become an sBTC Signer is determined through the community governance process of ratifying this SIP.
+
+sBTC Signers are responsible for accepting or rejecting all sBTC deposit and withdrawal operations submitted to the network. For a transaction to be fulfilled, at least 70% of the signers need to approve the transaction. This means that the liveness and reliability of the signers is crucial to the success of the protocol. The system is live ("resilient") if at least 70% of the sBTC Signer voting power are online and honest. Then (and only then), deposits and withdrawals happen in a timely manner. The system is safe ("trustworthy") if at least 30% of the sBTC Signer voting power is honest. Then, no theft of funds can occur.
+
+Throughout this release, sBTC will have an unchangeable and distinct signer set that will not explicitly be part of Stacks consensus. As a result, this release can activate without a hard fork. Each unique signer receives exactly one vote. Given there are 15 unique signers that comprise the system, it would require at least 11 out of 15 signatures for an sBTC operation to be fulfilled.
+
+While up to 30% of the signers can be offline without a user impact on the functioning of the protocol, it becomes more critical for the rest of the signers to approve sBTC operations because operations necessarily still need to meet 70% of the original signing power. If more than 30% of signers become unavailable, no sBTC operations will be approved because it will be impossible to get 70% approval when less than 70% are online. An operation that isn’t approved will become spendable by the user after a predetermined timeout has elapsed, measured in Bitcoin blocks.
+
+### sBTC Signer Responsibilities
+
+Overview of tasks the sBTC signers carry out:
+    - They react to requests to deposit BTC.
+    - They react to requests to withdraw BTC.
+    - They stay online often enough that deposits and withdrawals happen in a timely manner.
+    - They have to keep their signer hosts secure, so their keys don't get stolen.
+    - They have to proactively re-key every so often, which means moving the BTC to a new UTXO.
+- Signers must consolidate BTC as it is deposited.
```

Comment:
Nit: instead of linking to issues, can you put a summary of the mechanism in an appendix?  SIPs should be as complete as can be, so that as long as someone has a copy of the SIP (and only the SIP), they can develop a comprehensive understanding of the proposal without needing to do additional research.

Reply:
d5b1ac0750e2d5c5690f87a8f5473ad755019545